src/test/bin/quic_gtest.cpp

```diff
@@ -280,10 +280,11 @@ TEST_P(WithValidateStreamEventArgs, ValidateStreamEvents) {
 
 TEST(ParameterValidation, ValidateDesiredVersionSettings) {
     TestLogger Logger("QuicTestDesiredVersionSettings");
+    // TODO Fix these tests
```

Comment:
Just putting a comment here to make sure this isn't forgotten.

Reply:
Makes sense. This is the one I need to discuss the API scope with you and Anthony, and if VN is going to change any more. As I don't want to rewrite this, just to have it rewritten again in a week.